brave/brave-core
# Do not call eth_chainId for other than ETH networks

Resolves https://github.com/brave/brave-browser/issues/20703

- Added check for decimals number for a network to skip validation for non-ETH networks

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Try to add Filecoin network via network dialog in the settings
- Set decimals as 461
- No errors expected
